duckduckgo/Android
# Add the possibility to install debug builds alongside release builds

Task/Issue URL: https://app.asana.com/0/1125189844152671/1148296132081505

**Description**:
Currently, both debug and release builds share the same application package ID, meaning only one can be installed on a device at a time.

**Steps to test this PR**:
1. Create a release build and install.
1. Create a debug build and install.
1. Both builds should be able to live together and the debug build icon should be different.

**Work Manager still works**
1. After installing both release and debug, launch the apps and notice how the app doesn't crash because WorkManager was already initialized.

**Widget opens the correct app**
1. After install both apps, add a widget for each of them.
2. Each widget should open its own app without any issues.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
